hail/src/main/scala/is/hail/variant/Call.scala

```diff
@@ -43,6 +43,19 @@ object Call2 {
 
     Call(alleleRepr, phased, ploidy = 2)
   }
+
+  def withErrorID(aj: Int, ak: Int, phased: Boolean, errorID: Int): Call = {
+    if (aj < 0 || ak < 0)
+      fatal(s"allele indices must be >= 0. Found j=$aj and k=$ak.")
```

Comment:
Should this be tagged with the errorID too?

Reply:
yep, good catch